doc/changelog.md

```diff
@@ -92,6 +92,9 @@
   annotations.
   [(#751)](https://github.com/PennyLaneAI/catalyst/pull/751)
 
+* The installation guide page for Catalyst now includes a new summary section, allowing users to easily see together all the commands to run depending on their OS.
+  [(#756)](https://github.com/PennyLaneAI/catalyst/pull/756)
+
```

Comment:
Since this concerns the website only and not the distributed package I don't think it should have a changelog
```suggestion
```